test/Faker/Provider/InternetTest.php

```diff
@@ -117,4 +117,50 @@ public function testMacAddress()
     {
         $this->assertRegExp('/^([0-9A-F]{2}[:]){5}([0-9A-F]{2})$/i', Internet::macAddress());
     }
+
+    public function test_htmlLoremDefault()
+    {
+        $html = Internet::html();
+        $this->assertTrue($this->containsHtml($html));
+    }
+
+    public function test_htmlLoremSmallLengths()
+    {
+        $plus5 = Internet::html(Internet::SMALLEST_HTML_LENGTH+5);
+        $this->assertTrue($this->containsHtml($plus5));
+    }
+
+    public function test_htmlTooSmall()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        Internet::html(Internet::SMALLEST_HTML_LENGTH + 4);
+        Internet::html(Internet::SMALLEST_HTML_LENGTH - 1);
+    }
+
+    public function test_htmlMinimal()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        Internet::html(Internet::SMALLEST_HTML_LENGTH - 1);
+    }
+
+    public function test_htmlFibonacci()
+    {
+        $fiboLength = 22; // Fibonacci's sequence 22th number is 17711
+        $fibos= array(0, 1);
+        for ($i=2; $i<=$fiboLength; ++$i) {
+            $fibos[$i] = $fibos[$i-1] + $fibos[$i-2];
+        }
+        foreach($fibos as $fibo) {
+            if(9 < $fibo) {
+                $html = Internet::html($fibo);
+                $this->assertTrue($this->containsHtml($html));
+            }
+        }
+
+    }
+
+    protected function containsHtml($string)
+    {
+        return preg_match("/<[^<]+>/",$string, $matches) !== 0;
```

Comment:
That's right. Well, the purpose here was only checking that `Internet::html` actually return some html tags, rather  _validate_ any HTML output. I may remove this func, in order not to measled people.